api/capi/src/nnstreamer-capi-single-new.c

```diff
@@ -198,17 +198,86 @@ invoke_thread (void *arg)
   return NULL;
 }
 
+/**
+ * @brief Gets the tensors info from tensor-filter.
+ */
+static void
+ml_single_get_tensors_info_from_filter (GTensorFilterSingle * filter, gboolean is_input,
+    ml_tensors_info_h * info)
+{
+  ml_tensors_info_s *input_info;
+  GstTensorsInfo gst_info;
+  gchar *val;
+  guint rank;
+  gchar *prop_prefix, *prop_name;
+
+  if (is_input)
+    prop_prefix = g_strdup ("input");
+  else
+    prop_prefix = g_strdup ("output");
```

Comment:
These better be a static constant.
E.g., 
```
static const inputstr[] = "input";
static const outputstr[] = "ouput";
...
  prop_prefix = inputstr;
```

Reply:
Updated with static string.